include/flatbuffers/flatbuffers.h

```diff
@@ -1607,9 +1607,42 @@ inline bool BufferHasIdentifier(const void *buf, const char *identifier) {
                  identifier, FlatBufferBuilder::kFileIdentifierLength) == 0;
 }
 
-// Helper class to verify the integrity of a FlatBuffer
+/// @class Verifier
+/// @brief Helper class to verify the integrity of a FlatBuffer
+///
+/// The `Verifier` checks offsets within the data to ensure they are
+/// consistent with a correctly created buffer. Additionally it sets 
+/// conservative fixed limits for the maximum number of tables a
+/// flatbuffer can hold and the maximum depth of those tables,
+/// which can be increased if your application requires it.
+///
+/// For an example of these limits, consider a table `PriceTimeline`
```

Comment:
Thanks, but these examples seem a bit out of place. In-line documentation should be short, to the point, and non-redundant (no need to remind people of facts stated elsewhere in this file). More extensive explanation and examples should instead go into the .md files.

Reply:
I totally agree `DRY` (Don't repeat yourself) is an excellent principle for the code itself. Please consider that it is equally worthwhile to help users find the critical information they need where & when they need it. In other words if someone makes a mistake, the code and error messages should try to solve the problem for them.

Keras is a library makes for an excellent documentation example:
 - design philosophy: https://blog.keras.io/user-experience-design-for-apis.html
 - Keras documentation: https://keras.io/

I added this comment to this specific location because it is at exactly where I could have fixed my original problem. If this comment had been here it would have saved me several days of time. :-)

Reply:
Sure, but you should also appreciate that the problems people may encounter with FlatBuffers vary wildly based on their background, and we can't have `flatbuffers.h` turn into a tutorial for all sorts of cases that may not be relevant to the current reader. Hence my original request.